src/Schema/Events/Search.ts

```diff
@@ -0,0 +1,89 @@
+import { ActionType } from "."
+import { ContextModule } from "../Values/ContextModule"
+import { OwnerType, PageOwnerType } from "../Values/OwnerType"
+
+/**
+ * Schemas describing Search events
+ * @packageDocumentation
+ */
+
+/**
+ * A user focuses on a search box
+ *
+ * This schema describes events sent to Segment from [[FocusedOnSearchInput]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "focusedOnSearchInput",
+ *    context_module: "header",
+ *    context_page_owner_type: "home"
+ *  }
+ * ```
+ */
+export interface FocusedOnSearchInput {
+  action: ActionType.focusedOnSearchInput
+  context_module: ContextModule
+  context_page_owner_type: PageOwnerType
```

Comment:
Would recommend using generic `context_owner_thing` over page syntax here so we can share these events with iOs.